packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/components/Controls/components/Label/Label.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { SpacedText } from '@zooniverse/react-components'
+import getDataSeriesColor from '../../../../../../helpers/getDataSeriesColor'
+import getDataSeriesSymbol from '../../../../../../helpers/getDataSeriesSymbol'
+
+const StyledLabel = styled.span`
+  align-items: center;
+  cursor: pointer;
+  display: inline-flex;
+  flex-direction: row;
+  padding: 2.5px;
+
+  /* Could use Grommet gap prop instead */
+  svg {
+    padding: 0px 0.5ch;
+  }
+
+  /* Might remove this */
+  &:hover, &:focus-within {
+    background-color: rgba(216,216,216,0.4);
+  }
```

Comment:
What would be the reason for removing this? I suppose this comment should have more info if it stays, or either the styling or comment could be removed.

Reply:
I left a note for myself in case @beckyrother wanted it removed since the styles for these controls had to change. She okayed the background color, so I'll leave it in and remove the comment.